src/chess/model/Pawn.java

```diff
@@ -111,10 +141,20 @@ public void setNewPiece(Piece newPiece) {
             } else {
                 if (position.substring(1, 2).equals("7")) {
                     if (board[indexNumber - 1][indexLetter].isEmpty() &&
-                            board[indexNumber - 2][indexLetter].isEmpty()) {
+                        board[indexNumber - 2][indexLetter].isEmpty()) {
                         possibilities.add(position.substring(0, 1) + 5);
                     }
                 }
+                if (position.substring(1, 2).equals("4")) {
+                    if (canCaptureEnPassant(position, board)) {
+                        if (!board[indexNumber][indexLetter + 1].isEmpty()){
+                            possibilities.add(ControlGame.numToLetter(indexLetter + 1) + (indexNumber));
+                        }
```

Comment:
I want to set null after pawn move to remove the opponent's pawn, but I don't know which part I should put this action.
`board[indexNumber][indexLetter + 1].setPiece(null);`